services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -11,15 +13,37 @@ import scala.concurrent.{ ExecutionContext, Future }
 class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
     extends ProjectVersioning[VersioningException] {
 
+  private def getUpdateFileJson(projectFile: ProjectFile): String =
+    s"""{"branch":"master","content":"${projectFile.content}"}"""
+
+  private def getCreateFileJson(projectFile: ProjectFile): String =
+    s"""{"branch":"master","content":"${projectFile.content}","commit_message":"Init commit"}"""
```

Comment:
Oh wow.
You must never hardcode the body of the json in your code. You must write a case class for such a purpose. Something like
```
final case class GiveItAName(branch: String, content: String, commitMessage: Option[String])
```
And then write a `formattter` for it.

Reply:
Also, I see that @IaroslavTavchenkov uses `master` branch too and got a comment about it from @ilyaonishenko.
Since your PR contains the same [problem](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/59/files#r414009028), you must discuss it with @IaroslavTavchenkov and implement the same solution for it.

Reply:
Will discuss it, thanks